frontend/src/employee-mobile-frontend/components/attendances/AttendanceList.tsx

```diff
@@ -50,6 +50,12 @@ export default React.memo(function AttendanceList({
     )
   }
 
+  function getGroupIdByChild(child: AttendanceChild) {
+    return attendanceChildren
+      .map((ac) => ac.groupId)
+      .find((gId) => gId === child.groupId)
```

Comment:
Nit: No reason to loop through the list twice:
```suggestion
    return attendanceChildren.find((ac) => ac.groupId === child.groupId)?.groupId
```